js/ui.js

```
@@ -2898,6 +2930,21 @@
   });
 };
 
+const handleModelChange = (model, reload = true) => {
+  
+  STATE.analysisDone = false;
+  modelSettingsDisplay();
+  DOM.customListFile.value = config.customListFile[config.model];
+  DOM.customListFile.value
+    ? (LIST_MAP = i18n.get(i18n.LIST_MAP))
+    : delete LIST_MAP.custom;
+  document.getElementById(config[config.model].backend).checked = true;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)